Campus-iOS/PushNotifications/CryptoExportImportManager.swift

```diff
@@ -0,0 +1,287 @@
+//
+//  CryptoExportImportManager
```

Comment:
Isn't there a separate library for that?

Reply:
If not, create your own package? :D

Reply:
https://github.com/DigitalLeaves/CryptoExportImportManager

Its "meant" to be copied into the project. Not sure if its really necessary to create an own package for that...

Reply:
Yes sure, but long-term this copy-paste solution may lead to problems if Apple deprecates some APIs etc.
Isn't a requirement to change it, I just think there are cleaner ways to do it